src/common/midi_event.py

```diff
@@ -5,7 +5,8 @@
 @dataclass(order=True)
 class MidiEvent:
     event: mido.Message=field(compare=False) # Don't use this when sorting in a priority queue
-    timestamp: float # Time.time() returns seconds as float
+    timestamp: float                         # Time.time() returns seconds as float
+    play_note: bool = False                          # If false, don't send the note

```

Comment:
ah so this is how you do defaults with a dataclass. TIL

Reply:
Yes, did you need me to change it?

Reply:
oh sorry no, just thought it was cool lol